examples/menubar/menubar-1/mb-outreach.html

```diff
@@ -18,7 +18,7 @@ <h1 id="outreach">Outreach</h1>
         <h2 id="comm-engage">Community Engagement</h2>
         <p><a href="menubar-1.html#code-ex-1">Back to <code>menubar</code> example</a></p>
 
-        <h2 id="copr-rel">Corporate Relations</h2>
+        <h2 id="corp-rel">Corporate Relations</h2>
```

Comment:
I'm not sure idrefs should be spellchecked.

Reply:
They can be indvidually ignored, but in this case there are no refs https://github.com/w3c/aria-practices/search?q=copr&unscoped_q=copr and it looked like a typo

Reply:
looks like we have some unnecessary files left over from old content in the menubar-1 folder.

Reply:
Agree - looks like we can remove mb-research.html, mb-arts.html and mb-outreach.html

Reply:
Done in commit c57878b